app/controllers/admin/commands_controller.rb

```diff
@@ -51,7 +51,7 @@ def destroy
   private
 
   def command_params
-    params.require(:command).permit(:command, :project_id)
+    params.require(:command).permit(:name, :command, :project_id)
```

Comment:
how about this: if the first line is a comment -> use that as name
so inline editing still works nicely + compiled command makes sense